match2/wikis/starcraft2/match_legacy.lua

```diff
@@ -119,6 +119,9 @@ function p.convertParameters(match2)
 
 		if match.resulttype == 'default' then
 			match.resulttype = string.upper(match.walkover or '')
+			if match.resulttype == 'L' then
+				match.resulttype = 'unk'
```

Comment:
It would be nice to reduce our dependence on string literals for result types, and instead use constants, a module, etc

Reply:
`local _UNKNOWN_DEFAULT_LOSS = 'L'`?